.github/workflows/docker-image.yml

```
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "master" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)